.github/workflows/reusable-windows-msi.yml

```diff
@@ -0,0 +1,24 @@
+name: TestsMSI
+
+on:
+  workflow_call:
+    inputs:
+      arch:
+        description: CPU architecture
+        required: true
+        type: string
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    name: installer for ${{ inputs.arch }}
```

Comment:
I don't know if the name will be qualified in some other way?
```suggestion
    name: Windows installer for ${{ inputs.arch }}
```

Reply:
It's already prefixed with "windows" on the calling side. I intentionally didn't add it here. The UI side is fine.

Reply:
![Screenshot_2024-07-24-10-43-39-56_40deb401b9ffe8e1df2f1cc5ba480b12.jpg](https://github.![Screenshot_2024-07-24-10-41-25-10_320a9a695de7cdce83ed5281148d6f19.jpg](https://github.com/user-attachments/assets/a2bc6985-8f56-44ef-a0da-80c21e7860bb)

_2024-07-24-10-41-25-10_320a9a695de7cdce83ed5281148d6f19.jpg …]()

![Screenshot_2024-07-24-10-42-35-22_40deb401b9ffe8e1df2f1cc5ba480b12.jpg](https://github.com/user-attachments/assets/ccc0b13d-9733-4412-9bbe-c34f8686b11f)

Reply:
On the screenshots above, it's visible that it's also consistent with other reusable workflows.